scripts/security/run_openfdd_security_scan.sh

```
@@ -0,0 +1,236 @@
+  echo "" >>"$NMAP_FINDINGS"
+  echo "=== Port state lines ===" >>"$NMAP_FINDINGS"
+  grep -E '^[0-9]+/tcp' "$NMAP_SERVICES" >>"$NMAP_FINDINGS" 2>/dev/null || true
+fi
+
+cat >"$ZAP_FINDINGS" <<EOF
+ZAP findings summary
+Generated: $(date '+%Y-%m-%d %H:%M:%S')
+Target: $TARGET_URL
+EOF
+if [[ -f "${BASE_DIR}/${ZAP_MD}" ]]; then
+  grep -iE 'WARN|FAIL|High|Medium|Low|CSP|X-Frame|Header' "${BASE_DIR}/${ZAP_MD}" >>"$ZAP_FINDINGS" 2>/dev/null | head -80 || true
```

Shellcheck (0.11.0)

[error] This redirection overrides the output pipe. Use 'tee' to output to both.

(SC2260)